wiki/Game_mode/id.md

```diff
@@ -0,0 +1,46 @@
+
+Secara *default*, kamu akan disambut dengan mode permainan yang terakhir kali kamu mainkan setiap kali kamu membuka klien osu! milikmu. Untuk dapat berpindah dari satu mode permainan ke mode permainan lainnya, kamu dapat menggunakan tombol `Mode` yang berada pada bagian bawah layar Song Selection atau menekan kombinasi tombol `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, atau `Ctrl`+`4` untuk berpindah ke mode osu!, osu!taiko, osu!catch, atau osu!mania.
+
+![Tampilan antarmuka layar pemilihan mode permainan](/wiki/shared/Interface_mode.png "Tampilan antarmuka layar pemilihan mode permainan")
+
+## ![][o!s] osu!
+
+*Laman utama: [osu!](osu!)*
+
+*osu!* (terkadang juga disebut dengan *osu!standard*) merupakan mode permainan utama yang dimiliki oleh osu!. Mode permainan ini didasarkan pada permainan *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* yang dirilis oleh [iNiS](https://en.wikipedia.org/wiki/INiS "Wikipedia"). Dalam mode permainan ini, kamu dituntut untuk dapat mengenai semua circle, slider, dan spinner yang ada mengikuti irama lagu.
+
+![Tampilan permainan osu!](/wiki/shared/osu-gameplay.jpg "Permainan osu!")
+
+## ![][o!t] osu!taiko
+
+*Laman utama: [osu!taiko](osu!taiko)*
+
+*osu!taiko* merupakan mode permainan yang didasarkan pada seri permainan *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "Wikipedia")* yang dirilis oleh [Bandai Namco Entertainment](https://en.wikipedia.org/wiki/Bandai_Namco_Entertainment "Wikipedia"). Dalam mode permainan yang dibuat untuk menyerupai permainan drum ini, kamu dituntut untuk dapat mengenai not-not yang muncul pada layar sesuai dengan indikator warnanya masing-masing. 
+
+![Tampilan permainan osu!taiko](/wiki/shared/taiko-gameplay.jpg "Permainan osu!taiko")
+
+## ![][o!c] osu!catch
+
+*Laman utama: [osu!catch](osu!catch)*
+
+*osu!catch* merupakan mode permainan yang didasari pada *minigame EZ2CATCH* yang dipopulerkan oleh seri permainan [EZ2DJ](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia") dan beberapa simulator [Beatmania](https://en.wikipedia.org/wiki/Beatmania "Wikipedia") lainnya. Dalam mode permainan ini, kamu dituntut untuk dapat menangkap buah-buah yang jatuh dari bagian atas layar permainan dengan “*catcher*” yang kamu miliki.
+
+![Tampilan permainan osu!catch](/wiki/shared/catch-gameplay.jpg "Permainan osu!catch")
+
+## ![][o!m] osu!mania
+
+*Laman utama: [osu!mania](osu!mania)*
+
+*osu!mania* merupakan mode permainan yang didasari pada konsep permainan VSRG (vertical-scrolling rhythm game) pada umumnya, seperti [Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia") dan Beatmania. Dalam mode permainan yang pada awalnya dirancang oleh [woc2006](https://osu.ppy.sh/users/1105845) ini, kamu dituntut untuk dapat menekan tombol-tombol pada keyboard milikmu sesuai dengan laju perguliran not-not yang ada pada layar permainan.
```

Comment:
```suggestion
*osu!mania* merupakan mode permainan yang didasari pada konsep permainan VSRG (vertical-scrolling rhythm game) pada umumnya, seperti [Dance Dance Revolution](https://en.wikipedia.org/wiki/Dance_Dance_Revolution "Wikipedia") dan Beatmania. Dalam mode permainan yang pada awalnya dirancang oleh [woc2006](https://osu.ppy.sh/users/1105845), kamu dituntut untuk dapat menekan tombol-tombol pada keyboard sesuai dengan laju perguliran note yang ada pada layar permainan.
```

Reply:
fix efficiently word

Reply:
the `ini` in the sentence is actually required as it's the equivalent of the word `this` in this following sentence : `in this game mode that was first being developed by...`; removing the `ini` would turn the sentence over `in game mode that was...` which isn't correct grammatically :(

changed everything else otherwise